package.json

```diff
@@ -211,7 +211,7 @@
     "@material-ui/core": ">=4.0.0",
     "@material-ui/icons": ">=4.0.0",
     "@material-ui/styles": ">=4.0.0",
-    "ol": ">=6.5.0",
+    "ol": ">=6.9.0",
```

Comment:
It might be if there is a breaking change in ol, but we need to keep up with new versions in IAs.  If someone wants to use the latest supported ol-kit version they should be ok with using latest supported ol version as well.

Reply:
@gabe647 this current paradigm (`>=6.5.0`) will work for any `ol` minor version an IA would like to upgrade to going forward...there's no need to set a new threshold

Reply:
We don't have the bandwidth to support multiple versions of ol, what if we pinned the geokit peerDependency of ol to 6.9?  There shouldn't be any reason to use less than the latest supported version for IAs.

Reply:
changes in geokit im much more on board with since there's about to be a major/breaking change...but again the current peer dep there also allows many versions of openlayers and the onus to upgrade is on the implementer not the framework

Reply:
OK so I'll keep "ol": ">=6.5.0" for now and we can revisit this later as part of some major ol-kit version update.
Closing this PR now.